doc/concepts/promotion.rst

```diff
@@ -1,6 +1,20 @@
+=====================
 Diffing and Promotion
 =====================
 
+Dune writes its build output to a separate directory from your source code tree 
+(usually ``_build/<profile_name>``). You can use Diffing and Promotion flows 
+to compare your build output with your source tree, and/or copy the result of 
+rule output into your source tree to store the changes.
+
+Diffing
+=======
+
+You can use the ``(diff <file1> <file2>)`` directive in a rule to compare
+the output of the rule with a copy in your source tree. It is useful when
```

Comment:
```suggestion
its output with a copy in your source tree. It is useful when
```
Would this work to take out this one prepositional phrase? Reads a little smoother?